mathml-core/src/main/java/com/formulasearchengine/mathmltools/mml/MathDoc.java

```diff
@@ -163,10 +163,10 @@ public static String extractNamespacePrefix(StringBuffer rawXml) {
         return null;
     }
 
-    public static DOMInputImpl getMathMLSchema() {
+    public static LSInput getMathMLSchema() {
         SchemaInput schemaInput = new SchemaInput().invoke();
         InputSource inputSource = schemaInput.getInputSource();
-        final DOMInputImpl input = new DOMInputImpl();
+        final LSInput input = new DOMInputImpl();
```

Comment:
Here I got stuck. No Idea how to get a new DOMInputImpl. @AndreG-P @vstange

Reply:
Is this maybe something you are looking for:
[DOMImplementationLS](https://docs.oracle.com/en/java/javase/11/docs/api/java.xml/org/w3c/dom/ls/DOMImplementationLS.html)
``` java 
final LSInput input = DOMImplementationLS.createLSInput();
```